packages/website/src/features/Packages/FunctionInput.tsx

```diff
@@ -90,6 +91,14 @@ export const FunctionInput: FC<Props> = ({
       case input.type.startsWith('tuple'):
         // TODO: implement value prop for TupleInput
         return <TupleInput input={input} handleUpdate={_handleUpdate} />;
+      case input.type.startsWith('bytes32'):
```

Comment:
please add handlers for other bytes types too (bytes1, bytes2, bytes4, bytes8, bytes16)

Reply:
Added other types.